site/_posts/2020-06-05-private-labeling-with-fusionauth.md

```diff
@@ -1,31 +1,31 @@
 ---
 layout: blog-post
 title: Tenants and private labeling
```

Comment:
What if we changed the title to "Tenant APIs and Private Labeling for Multi-tenant Applications"? Does that even work?

Reply:
How about "How FusionAuth Can Help With Multi-Tenant Private Label Applications"? 

Or "Why You Might Want to Build A Multi-Tenant Private Label App?"

"How You Can Use FusionAuth's Multi-Tenant Feature To Create A Private Label Offering".

Reply:
I like "How To Use FusionAuth's Multi-Tenant Feature To Create A Private Label Offering"